java-frontend/src/test/files/se/FlowComputation.java

```diff
@@ -48,5 +48,13 @@ void invocation_target(Object a) {
     }
   }
 
+  void test(Object a, Object b) {
+    if (a == b) { // flow@catof
```

Comment:
Sorry, this is not relevant to this ticket and should be indeed part of SONARJAVA-2303, I removed it from this PR